### PR TITLE
Setup cli

### PR DESCRIPTION
**Pull request by Selchuk-Karakus**
Title: 
Create CLI command to list members #4 

Description
A function that prints the member data to the console displaying key information.

Write a summary of your change. Explain why you have made it.
We needed to provide a way for the members to access the member's data we stored.

Related to
#4 

Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [x] I have commented my code
- [x] I have updated any documentation

### CODE REVIEW

**Comment by Selchuk-Karakus** [comment]
@craig-dsilva Could you please review this PR. Thank you.

File: cli.js [hunk 1]
```diff
@@ -22,5 +22,14 @@ if (command === "list") {
 
 function runListCommand() {
   // Replace this with your implementation
-  console.log("Called the list command");
+  members.forEach(({firstName, lastName, jobTitle, location, userSlack}) =>
```
Comment (craig-dsilva) [inline cli.js:25]: Nice way to extract object keys

**Comment by craig-dsilva** [comment]
Well done

File: cli.js (hunk 1 above)
Comment (Selchuk-Karakus) [reply cli.js:25]: @craig-dsilva Thanks for taking the time to do this for me.

**Pull request merged**